sycl/test/abi/layout_vec.cpp

```diff
@@ -0,0 +1,22 @@
+// RUN: %clangxx -fsycl -c -fno-color-diagnostics -Xclang -fdump-record-layouts %s | FileCheck %s
+// REQUIRES: linux
+
+// clang-format off
+
+#include <CL/sycl/types.hpp>
+
+void foo(sycl::vec<int, 4>) {}
```

Comment:
Is looking at specializations of lengths of four and sixteen enough for testing? AFAIK {{vec}} supports 2, 3, 4, 8, 16 as size.